pyNTCIREVAL/metrics/recall.py

```diff
@@ -0,0 +1,11 @@
+from .metric import Metric
+
+class Recall(Metric):
+    def __init__(self, total_positive):
```

Comment:
Better to take `xrelnum` as input for `__init__` and compute the `total_positive` in `__init__`. This would enable the users to take the same actions for all kinds of metrics.